### ISSUE TITLE
read.csv treat column as character instead of number if missing value after valid value

### ISSUE THREAD

**Issue by comicfans** [oracle/fastr#42]
read following csv
```csv
col1,col2
,1
2,
```
with 
```R
csv <-read.csv('test.csv',stringsAsFactors=F)
typeof(csv$col1)
csv$col1
typeof(csv$col2)
csv$col2
```
GNU-R
```R
integer
NA 2
integer
1 NA
```

fastR
```R
integer
NA 2
character
"1"  ""
```

seems this only happen if missing value after valid value. for example add more row like this:
```csv
col1,col2
,1
2,
,3
```
fastR typeof col1 will also changed to character

**Comment by steve-s** [comment]
Hi comicfans,

thanks for the report, we can reproduce and are working on a fix.

**Comment by steve-s** [comment]
Should be fixed in ce2ee00e011fd0e888ba52b283ca64d7db900095
